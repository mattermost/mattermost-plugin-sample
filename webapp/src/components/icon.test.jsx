import React from 'react';
import {shallow} from 'enzyme';

import Icon from './icon';

test('icon test', () => {
    const wrapper = shallow(<Icon/>);
    expect(wrapper).toMatchSnapshot();
});
