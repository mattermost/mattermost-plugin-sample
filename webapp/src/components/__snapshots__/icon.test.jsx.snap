// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon test 1`] = `ShallowWrapper {}`;
